api/v1beta1/disruption_types.go

```diff
@@ -187,6 +189,10 @@ type DisruptionStatus struct {
 	DesiredTargetsCount int `json:"desiredTargetsCount"`
 }
 
+type DisruptionFilter struct {
+	Annotations labels.Set `json:"annotation,omitempty"`
```

Comment:
```suggestion
	Annotations labels.Set `json:"annotations,omitempty"`
```

Sorry I didn't catch this one before :/
We should use `annotations` (plural) which is the same writing done in Kubernetes resources. You may want to adapt examples too (both complete and the specific one).